tests/e2e/matmul/generate_e2e_matmul_tests.py

```diff
@@ -137,7 +137,7 @@ def get_test_shapes(shapes_id: ShapesId):
         # (see get_test_generators).
     ]
   if shapes_id == ShapesId.GPU_LARGE:
-    return [TestShape(m=256, k=128, n=512)]
```

Comment:
why do we need this change?

Reply:
I have all my notes and debugging pictures for 512x256x128. It is fine to revert it to the original, but you might see the m and n flipped again in some time the next PR :)